modules/local/multiqc.nf

```diff
@@ -38,10 +28,13 @@ process MULTIQC {
     path "*_data"              , emit: data
     path "*_plots"             , optional:true, emit: plots
 
+    when:
+    task.ext.when == null || task.ext.when
+
     script:
-    def software      = getSoftwareName(task.process)
+    def args = task.ext.args ?: ''
     def custom_config = params.multiqc_config ? "--config $multiqc_custom_config" : ''
     """
-    multiqc -f $options.args $custom_config .
+    multiqc -f $args $custom_config .
     """
 }
```

Comment:
why not using the multiQC nf-core module here ?

Reply:
There didnt used to be one as it was a custom module everytime but they seem to have made one now!